nemo_skills/inference/model/vllm_multimodal.py

```
@@ -108,3 +148,298 @@ def _process_audio_response(self, audio_data, response_id: str) -> dict:
+            Message dict with content converted to list format including audio.
+        """
+        if "audio" not in message and "audios" not in message:
+            return message
+
+        content = message.get("content", "")
+        if isinstance(content, str):
+            message["content"] = [{"type": "text", "text": content}]
+        elif isinstance(content, list):
+            message["content"] = content
+        else:
+            raise TypeError(f"Unexpected content type: {type(content)}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)